jobclient/python/docs/source/api.rst

```diff
@@ -42,6 +42,33 @@ this may be overridden in the ``JobClient.submit`` method.
 .. autoclass:: Application
    :members:
 
+Containers
+^^^^^^^^^^
+
+By default, jobs submitted to Cook will use the ``python:3.6`` container, which
```

Comment:
This actually depends on how Cook Scheduler is configured, so I don't think we should say this as a blanket statement.